action.yml

```
@@ -4,21 +4,55 @@ author: "Łukasz Langa and contributors to Black"
 inputs:
   options:
     description:
-      "Options passed to black. Use `black --help` to see available options. Default:
+      "Options passed to Black. Use `black --help` to see available options. Default:
       '--check'"
     required: false
     default: "--check --diff"
   src:
-    description: "Source to run black. Default: '.'"
+    description: "Source to run Black. Default: '.'"
     required: false
     default: "."
   black_args:
     description: "[DEPRECATED] Black input arguments."
     required: false
     default: ""
+  version:
+    description: 'Python Version specifier (PEP440) - e.g. "21.5b1"'
+    required: false
+    default: ""
 branding:
   color: "black"
   icon: "check-circle"
 runs:
-  using: "docker"
-  image: "action/Dockerfile"
+  using: composite
+  steps:
+    - run: |
+        # Exists since using github.action_path + path to main script doesn't work because bash
+        # interprets the bashslashes in github.action_path (which are used when the runner OS
```

```suggestion
        # interprets the backslashes in github.action_path (which are used when the runner OS
```
Or is this some new kind of slashes I don't know about